mozilla-release/browser/base/content/aboutDialog.xul

```diff
@@ -44,8 +44,6 @@
       <vbox id="rightBox" flex="1">
         <label id="distribution" class="text-blurb"/>
         <label id="distributionId" class="text-blurb"/>
-#expand <label id="version">&version.basedon;__MOZ_APP_VERSION__</label>
```

Comment:
It won't be added unless someone resolves a conflict in favor of upstream version.
So I'd say, it's pretty safe now.
Adding 2 extra lines, instead of deleting one doesn't seem like a good tradeoff.